### ISSUE TITLE
zombienet / statemine weirdness

### ISSUE THREAD

**Issue by ryanleecode** [paritytech/capi#730]
```
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
SyntaxError: Unterminated string in JSON at position 524288
    at JSON.parse (<anonymous>)
    at ZombienetProvider.network (file:///github/workspace/providers/frame/zombienet.ts:44:21)
    at async file:///github/workspace/providers/frame/zombienet.ts:32:14
Download http://localhost:4646/mod.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/alice/@latest/mod.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/alice/@latest/types/pallet_xcm/pallet.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/alice/@latest/types/rococo_runtime/mod.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/collator/@latest/mod.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/collator/@latest/types/cumulus_pallet_parachain_system/pallet.ts
Download http://localhost:4646/frame/zombienet/zombienets/statemine.toml/collator/@latest/types/statemine_runtime.ts
error: Import 'http://localhost:4646/frame/zombienet/zombienets/statemine.toml/alice/@latest/mod.ts' failed: 500 Internal Server Error
    at file:///github/workspace/examples/xcm_asset_teleportation.ts:2:34
```

Saw this in some logs... not sure if its a concern or not.

https://github.com/paritytech/capi/actions/runs/4380576960/jobs/7667795286

**Comment by kratico** [comment]
@pepoviola Would you know if the `zombie.json` file is modified more than once when the network is launched?

Right now, our logic watches for the first the filesystem change on `zombie.json` to assume that the network is up to start websocket connections

**Comment by tjjfvi** [comment]
I've fixed this in #641 by using the programmatic API instead of the CLI. Unfortunately, because of paritytech/zombienet#787, we have to run it in a subprocess, but once that's fixed, the zombienet logic will be greatly simplified.
